torch/distributions/kl.py

```diff
@@ -209,6 +210,11 @@ def _kl_gumbel_gumbel(p, q):
     return t1 + t2 + t3 - (1 + _euler_gamma)
 
 
+@register_kl(Geometric, Geometric)
+def _kl_geometric_geometric(p, q):
+    return -p.entropy() - torch.log1p(-q.probs) / p.probs - q.logits
```

Comment:
Just curious, where did you find a formula for this? or did you compute it by hand?

Reply:
my bad, updated the snippet